src/state-summary/state-card-lock.html

```diff
@@ -0,0 +1,42 @@
+<link rel="import" href="../../bower_components/polymer/polymer-element.html">
+
+<link rel="import" href="../../bower_components/iron-flex-layout/iron-flex-layout-classes.html">
+
+<link rel="import" href="../components/entity/state-info.html">
+<link rel="import" href="../components/entity/ha-entity-lock.html">
+
+<dom-module id="state-card-lock">
+  <template>
+    <style include="iron-flex iron-flex-alignment"></style>
+    <style>
+      ha-entity-lock {
+        margin: -4px -16px -4px 0;
+        padding: 4px 16px;
```

Comment:
This was used for toggle to increase the tap target to 48px.
Since ha-entity-lock doesn't have a single tap target - this is not needed